easy_switch_user/static/tests/tours/test_switch_user.js

```diff
@@ -0,0 +1,27 @@
+odoo.define("easy_switch_user.switch_user_tour", function (require) {
+    "use strict";
+
+    const tour = require("web_tour.tour");
+
+    tour.register(
+        "easy_switch_user.switch_user",
+        {
+            test: true,
+        },
+        [
+            {
+                content: "Check admin user name",
+                trigger: "span:contains(Mitchell Admin)",
```

Comment:
Please be more specific, there are a lot of span's on the HTML:
```suggestion
                trigger: "span.oe_topbar_name:contains(Mitchell Admin)",
```

Same below